ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -81,6 +81,26 @@ class ClientQueryViewController: UITableViewController, Themeable, UIDropInterac
 
 	private var _actionProgressHandler : ActionProgressHandler?
 
+	var currentPathContainsFolders : Bool {
+		let folders = items.filter { (item) -> Bool in
+			if item.type == .collection {
+				return true
+			}
+			return false
+		}
+		if folders.count == 0 {
+			return false
+		}
+		return true
```

Comment:
@hosy I would return `false` by default and I think check for `folders.count` is superflous, since if a folder is found, it would be caught by condition above